noisy_simulator/src/instrument.rs

```diff
@@ -102,7 +108,8 @@ fn summed_kraus_operators(operations: &[Operation]) -> Result<Vec<SquareMatrix>,
                 .iter()
                 .map(|k| {
                     // This code is doing the equivalent to:
-                    // choi_matrix += vec(K) * vec(K)^dagger
+                    // choi_matrix += vec(K) * vec(K*)
+                    // where * denotes the complex conjugate
```

Comment:
Nitpick: it seems a little odd to use `*` for both multiplication and conjugate. Another option to consider would be `Kdg` or `Kadj` for dagger and adjoint respectively. I've seen both used in a lot of code, and I tend toward the adjoint variation myself, though either would be fine.